packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -80,5 +81,15 @@ export async function prefetchLatestSnapshot(
             assert(cacheP !== undefined, "caching was not performed!");
             await cacheP;
             return true;
-    }).catch((error) => false);
+    }).catch(async (error) => {
+        const errorType = error.errorType;
```

Comment:
can we share this code (same code exists in fetchSnapshot) by moving it into fetchSnapshotWithRedeem?
Also, in both places could you please add check for typeof & not null?